docs/manual-installation.md

```diff
@@ -0,0 +1,128 @@
+# Manual installation tutorial
+
+## Step by step
+
+In order to develop on decidim, you'll need:
+
+* **Git** 2.15+
+* **PostgreSQL** 9.4+
+* **Ruby** 2.5.0 (2.3+ should work just fine, but that's the version we test against)
+* **NodeJS** 9.x.x
+* **ImageMagick**
+* **Chrome** browser and [chromedriver](https://sites.google.com/a/chromium.org/chromedriver/).
+
+We're starting with an Ubuntu 16.04 LTS. This is an opinionated guide and YMMV, so if you're free to use the technology that you fell most comfortable on. If you have any doubts and you're blocked you can go and ask on [our Gitter](https://gitter.im/decidim/decidim). We recommend that you follow some Ruby on Rails tutorials (like [Getting Started with Ruby on Rails](http://guides.rubyonrails.org/getting_started.html)) and have some knowledge on how gems and engines work.
+
+On this tutorial we'll see how to install rbenv, PostgreSQL and Decidim, and how to configure everything together.
+
+### Installing rbenv
+
+First we're going to install rbenv, for managing various ruby versions. We're following the guide from [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-16-04). You could also use [rvm](https://rvm.io/) as an alternative on this step.
+
+```bash
+sudo apt-get update
+sudo apt-get install -y build-essential autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+source ~/.bashrc
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+rbenv install 2.5.0
+rbenv global 2.5.0
+echo "gem: --no-document" > ~/.gemrc
+gem install bundler
+```
```

Comment:
This still looks too specific to me... For example, it's already outdated and it's now recommending an potentially insecure ruby version without all the latest security patches (2.5.1 is out).

I would probably remove these instructions and mention that you need an up to date ruby and point out to official docs on installing it (or maybe even to our docker image).

But if we want to keep it in the repo we need to commit to keeping it up to date, so we should get that version updated.